frontendGraph/src/main/java/com/virtuslab/gitmachete/frontend/graph/print/elements/IPrintElement.java

```diff
@@ -0,0 +1,14 @@
+package com.virtuslab.gitmachete.frontend.graph.print.elements;
+
+import org.checkerframework.checker.index.qual.NonNegative;
+
+public interface IPrintElement {
+
+  @NonNegative
+  int getRowIndex();
+
+  @NonNegative
+  int getPositionInCurrentRow();
```

Comment:
Hmm I'm a bit confused what `current` and `other` are supposed to mean in this package

Reply:
This allows for creating diagonal edges. The edge from the node at row index `r` and position `p` to the node at row index `r+1` and position `p+1` is a simply diagonal edge.  However, I will remove the `positionInOtherRow` since it is always equal to the current.